pkg/tests/api/alerting/api_prometheus_test.go

```diff
@@ -671,58 +672,16 @@ func TestPrometheusRulesPermissions(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, 200, resp.StatusCode)
 
-		require.JSONEq(t, `
-{
-	"status": "success",
-	"data": {
-		"groups": [{
-			"name": "arulegroup",
-			"file": "folder1",
-			"rules": [{
-				"state": "inactive",
-				"name": "rule under folder folder1",
-				"query": "[{\"refId\":\"A\",\"queryType\":\"\",\"relativeTimeRange\":{\"from\":18000,\"to\":10800},\"datasourceUid\":\"-100\",\"model\":{\"expression\":\"2 + 3 \\u003e 1\",\"intervalMs\":1000,\"maxDataPoints\":43200,\"type\":\"math\"}}]",
-				"duration": 120,
-				"annotations": {
-					"annotation1": "val1"
-				},
-				"labels": {
-					"label1": "val1"
-				},
-				"health": "ok",
-				"type": "alerting",
-				"lastEvaluation": "0001-01-01T00:00:00Z",
-				"evaluationTime": 0
-			}],
-			"interval": 60,
-			"lastEvaluation": "0001-01-01T00:00:00Z",
-			"evaluationTime": 0
-		},
-		{
-			"name": "arulegroup",
-			"file": "folder2",
-			"rules": [{
-				"state": "inactive",
-				"name": "rule under folder folder2",
-				"query": "[{\"refId\":\"A\",\"queryType\":\"\",\"relativeTimeRange\":{\"from\":18000,\"to\":10800},\"datasourceUid\":\"-100\",\"model\":{\"expression\":\"2 + 3 \\u003e 1\",\"intervalMs\":1000,\"maxDataPoints\":43200,\"type\":\"math\"}}]",
-				"duration": 120,
-				"annotations": {
-					"annotation1": "val1"
-				},
-				"labels": {
-					"label1": "val1"
-				},
-				"health": "ok",
-				"type": "alerting",
-				"lastEvaluation": "0001-01-01T00:00:00Z",
-				"evaluationTime": 0
-			}],
-			"interval": 60,
-			"lastEvaluation": "0001-01-01T00:00:00Z",
-			"evaluationTime": 0
-		}]
-	}
-}`, string(b))
+		body := asJson(t, b)
+		// Sort, for test consistency.
+		sort.Slice(body.Data.Groups, func(i, j int) bool { return body.Data.Groups[i].File < body.Data.Groups[j].File })
+		require.Equal(t, "success", body.Status)
+		// The request should see both groups, and all rules underneath.
```

Comment:
The test is just trying to make sure the right records are visible based on the authz settings. Assert only this and nothing more. (otherwise, adding new fields to rule groups would break this test, and that shouldn't matter)